$NetBSD: patch-src_sysdep_c,v 1.1 2012/06/01 03:21:51 makoto Exp $

To avoid the error starting following line:
  sysdep.c:2549:16: error: 'CTL_KERN' undeclared (first use in this function)

--- src/sysdep.c.orig	2012-06-01 12:05:10.000000000 +0900
+++ src/sysdep.c	2012-06-01 12:10:51.000000000 +0900
@@ -44,6 +44,10 @@
 #include <math.h>
 #endif
 
+#ifdef __NetBSD__
+#include <sys/sysctl.h>
+#endif
+
 #ifdef DARWIN_OS
 #include <sys/sysctl.h>
 #endif
